docs/source/quickstart.rst

```diff
@@ -15,10 +15,6 @@ You install MLflow by running:
         # Install MLflow
         pip install mlflow
 
-        # Install MLflow with the experimental MLflow Pipelines component
-        pip install mlflow[pipelines]  # for pip
-        conda install -c conda-forge mlflow-pipelines  # for conda
```

Comment:
Looks like some changes we made in https://github.com/mlflow/mlflow/pull/6768/ were ignored when we merged branch 2.0 on master.